### PR TITLE
feat(dup_enhancement#26): update duplicate rpc thrift in go-client

### PR DESCRIPTION
**Pull request by foreverneverer**
# Ref-Issue
https://github.com/apache/incubator-pegasus/issues/892

### CODE REVIEW

File: go-client/session/radmin_rpc_types.go [hunk 1]
```diff
@@ -15,7 +15,7 @@
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
-*/
+ */
```
Comment (acelyc111) [inline go-client/session/radmin_rpc_types.go:18]: revert it plz

File: go-client/session/admin_rpc_types.go [hunk 2]
```diff
@@ -15,7 +15,7 @@
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
-*/
+ */
```
Comment (acelyc111) [inline go-client/session/admin_rpc_types.go:18]: revert it plz

File: go-client/session/radmin_rpc_types.go (hunk 1 above)
Comment (foreverneverer) [reply go-client/session/radmin_rpc_types.go]: done

**Review by acelyc111 — APPROVED** [review]

**Comment by acelyc111** [comment]
We should unify the IDL *.thrift files, now they are distributed every where, we can do it after rdsn has been merged to Pegasus.

**Comment by foreverneverer** [comment]
> We should unify the IDL *.thrift files, now they are distributed everywhere, we can do it after rdsn has been merged to Pegasus.

Yeah, I have the same plan

**Review by hycdong — APPROVED** [review]

**Pull request merged**